compiler/rustc_const_eval/src/interpret/intrinsics.rs

```diff
@@ -830,6 +840,36 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
         interp_ok(())
     }
 
+    fn float_minimum_intrinsic<F>(
+        &mut self,
+        args: &[OpTy<'tcx, M::Provenance>],
+        dest: &MPlaceTy<'tcx, M::Provenance>,
+    ) -> InterpResult<'tcx, ()>
+    where
+        F: rustc_apfloat::Float + rustc_apfloat::FloatConvert<F> + Into<Scalar<M::Provenance>>,
+    {
+        let a: F = self.read_scalar(&args[0])?.to_float()?;
+        let b: F = self.read_scalar(&args[1])?.to_float()?;
+        let res = self.adjust_nan(a.minimum(b), &[a, b]);
```

Comment:
Please move the `minimum` call into its own `let`-binding -- it is the most important step in this function and currently it is very easy to miss. (Same for `maximum` below.)

(Yeah the existing methods have the same problem... sorry for that.)

Reply:
Sure, moved a binding.